sidebars.json

```diff
@@ -1,14 +1,27 @@
 {
   "someSidebar": {
-    "Getting Started": ["getting-started-introduction", "getting-started-contributing"],
+    "Getting Started": [
+      "getting-started-introduction",
+      "getting-started-contributing"
+    ],
     "New Developer Onboarding": [
       "onboarding-join-github-organization",
       "onboarding-dev-workflow",
       "onboarding-editor",
       "onboarding-git"
     ],
-    "Guides": ["guide-code-quality", "guide-devops", "guide-code-review"],
```

Comment:
lol not a fan of the one line style I see

Reply:
lol it just auto formatted